### PR TITLE
Feature/native upload

### PR DESCRIPTION
**Pull request by jacksonsouza**

### CODE REVIEW

File: lib/unimatrix/activist/resources/activity.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ class Activity < Base
     field    :subject_type
     field    :state
     field    :message
+    field    :realm
```
Comment (jaclynj) [inline lib/unimatrix/activist/resources/activity.rb]: this should be has_one :realm, not a field

File: lib/unimatrix/activist/resources/realm.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+module Unimatrix::Activist
+
+  class Realm < Base
+    field :id
```
Comment (jaclynj) [inline lib/unimatrix/activist/resources/realm.rb]: shouldnt it have a uuid?

File: lib/unimatrix/activist/resources/activity.rb (hunk 1 above)
Comment (sprestage) [reply lib/unimatrix/activist/resources/activity.rb]: Changed and successfully tested.

File: lib/unimatrix/activist/resources/realm.rb (hunk 2 above)
Comment (sprestage) [reply lib/unimatrix/activist/resources/realm.rb]: Changed and successfully tested.

**Pull request merged**